### ISSUE TITLE
[BUG]: `resizeMode` prop is not implemented on web 

### ISSUE THREAD

**Issue by gigobyte** [TheWidlarzGroup/react-native-video#4297]
### What happened?

The `object-fit` style property of the video element in [Video.web.tsx](https://github.com/TheWidlarzGroup/react-native-video/blob/d934f214f5ac309490f0cf3fd80927c8657e2af3/src/Video.web.tsx#L371) should respect the `resizeMode` prop.

Currently the `resizeMode` prop is not implemented on web, so setting it (e.g. to `"cover"`) has no effect on how the video is displayed.

### Steps to reproduce

1. Set `resizeMode` to `"cover"` on a `Video` component.
2. Run the app on web and observe the video in a browser.
3. Notice that the video does not respect the `resizeMode` value.

### react-native-video version

6.8.0

### Platforms

Web

### OS version

Chrome

### Device

Real device

### Architecture

Old Architecture

**Comment by freeboub** [comment]
@zoriya Can I ping you on web ticket ? :) not sure you receive a notifcation on all ticket and / or you have time to handle them ?! Thank you !

**Comment by zoriya** [comment]
Sure!

I wanted to wait for bun before starting a new PR to prevent a huge conflict.

**Comment by gigobyte** [comment]
The issue still exists.

For anyone looking for a quick fix, you can render something like this next to your `Video`, of course the CSS selector and value depend on your app:

```ts
function WebVideoObjectFitFix() {
  if (Platform.OS !== 'web') {
    return null
  }

  return <style>{'video { object-fit: cover !important; }'}</style>
}
```

**Comment by TWG-AI** [comment]
Thanks for the report! We've confirmed that `resizeMode` is not currently applied on the web platform — the prop value isn't being mapped to the corresponding CSS `object-fit` on the underlying `<video>` element.

We've tracked this internally. The fix is well-scoped to a single file, so we've marked this as a **good first issue** for anyone looking to contribute — it's a great entry point if you'd like to submit a PR.

> [!NOTE]
> This response was generated by TWG-AI, the TheWidlarzGroup automated triage assistant.

**Comment by huytdps13400** [comment]
I opened #5046 against `support/6.x.x`. It maps all four v6 `resizeMode` values to the corresponding CSS `object-fit` behavior and includes a built-output regression test covering `contain`, `cover`, `stretch`, and `none`.
